.travis.yml

```diff
@@ -50,6 +50,11 @@ matrix:
 before_script:
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
   - phpenv config-rm xdebug.ini || echo 'No xdebug config.'
+  - |
+    if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then
+      composer global require wp-coding-standards/wpcs phpcompatibility/phpcompatibility-wp dealerdirect/phpcodesniffer-composer-installer
+      phpcs
+    fi
```

Comment:
`before_script` shouldn't run any scripts.  It's better to have a separate job for phpcs.

Reply:
Done.